pkg/lockfile/parse-gradle-verification-metadata_test.go

```diff
@@ -0,0 +1,689 @@
+package lockfile_test
+
+import (
+	"io/fs"
+	"testing"
+
+	"github.com/google/osv-scanner/pkg/lockfile"
+)
+
+func TestGradleVerificationMetadataExtractor_ShouldExtract(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name string
```

Comment:
delete `name` if it is not used in `t.Run`

Reply:
it is being used though? `t.Run(tt.name, func(t *testing.T) {`